packages/frontend/src/components/common/ResourceList/ResourceActionMenu.tsx

```diff
@@ -75,6 +76,27 @@ const ResourceListActionMenu: FC<Props> = ({ item, spaces, url, onAction }) => {
                             });
                         }}
                     />
+                    {user.data?.ability.can('manage', 'Project') && (
+                        <MenuItem2
+                            role="menuitem"
+                            icon="pin"
+                            text={
+                                isPinned
+                                    ? 'Unpin from homepage'
+                                    : 'Pin to homepage'
+                            }
+                            onClick={(e) => {
+                                e.preventDefault();
+                                e.stopPropagation();
+
+                                setIsOpen(false);
+                                onAction({
+                                    type: ResourceListAction.PIN_TO_HOMEPAGE,
+                                    item,
+                                });
+                            }}
+                        />
+                    )}
```

Comment:
can either of you check if this is enough permission? @ZeRego @rephus

Reply:
This will only test if the user has permission to manage at least of project, you must also provide the project uuid to test a specific project

Reply:
I also think it should check on resource type rather than a much broader project permission scope

for charts:
```
can('manage', 'SavedChart', {
    organizationUuid:,
    projectUuid:,
})
```

for dashboard:
```
can('manage', 'Dashboard', {
    organizationUuid:,
    projectUuid:,
})
```

Reply:
```
.can('update', subject('Project', { projectUuid, OrganizationUuid }))
// or manage - but update is more specific
```

Reply:
@IrakliJani yup those checks but with `update` instead of `manage`